tests/closure/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -108,9 +108,11 @@ $longArgs_shortVars = function (
     // body
 };
 
-$shortArgs_longVars = function (
-    $arg
-) use ($longVar1, $longerVar2, $muchLongerVar3) {
+$shortArgs_longVars = function ($arg) use (
+    $longVar1,
+    $longerVar2,
+    $muchLongerVar3
+) {
```

Comment:
This change was not intentional, but i'm not sure if it's actually for the better?

Reply:
👍 agreed!